Confus/Confus/Game.h

```diff
@@ -55,6 +55,9 @@ namespace Confus
 		Player m_SecondPlayerNode;
         RespawnFloor m_RedRespawnFloor;
         RespawnFloor m_BlueRespawnFloor;
+
+        //use std::vector
```

Comment:
Added it as a reminder for myself, removed it in my current version for the next feature.